### PR TITLE
Add perf tests for simple operations involving BIG-IP

### PR DESCRIPTION
**Pull request by bmarshall13**
Add a simple benchmark test to measure how long it takes to apply() simple configurations of various sizes.

These tests will be skipped if there are no big-ip credentials defined.

### CODE REVIEW

File: test/f5_cccl/conftest.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+from f5.bigip import ManagementRoot
+
+from icontrol.exceptions import iControlUnexpectedHTTPError
+
+requests.packages.urllib3.disable_warnings()
+
+
+def _wrap_instrument(f, counters, name):
+    def instrumented(*args, **kwargs):
+        #pytest.set_trace()
```
Comment (edarzins) [inline test/f5_cccl/conftest.py]: Dead code, remove.

File: test/f5_cccl/conftest.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        bigip_fix = ManagementRoot(hostname, username, password)
+        bigip_fix = instrument_bigip(bigip_fix)
+    else:
+        bigip_fix = MagicMock()
+
+    yield bigip_fix
+
+@pytest.fixture(scope="function")
+def bigip_rest_counters(bigip):
+    #icr = mgmt_root.__dict__['_meta_data']['icr_session']
```
Comment (edarzins) [inline test/f5_cccl/conftest.py]: Dead code, remove.

File: test/f5_cccl/conftest.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+@pytest.fixture(scope="function")
+def cccl(bigip, partition):
+    cccl = F5CloudServiceManager(bigip, partition)
+    yield cccl
+    cccl.apply_config({})
+
+
+@pytest.fixture()
+def pool(bigip, partition):
```
Comment (edarzins) [inline test/f5_cccl/conftest.py]: I don't see that you use the "pool" fixture anywhere -- is it still necessary?

Comment (bmarshall13) [reply test/f5_cccl/conftest.py]: I was going to move these sharable fixtures all here, so instead, I'll remove the duplicates from test/f5_cccl/resource/ltm/policy/test_policy.py.

**Pull request merged**